spec/jobs/representation_order_creator_job_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+RSpec.describe RepresentationOrderCreatorJob, type: :job do
+  include ActiveJob::TestHelper
+  let(:argument_hash) do
+    {
+      offence_id: 'LONG-UUID',
+      status_code: 'ABC',
+      maat_reference: 'STRING',
+      effective_start_date: 'DATE',
+      effective_end_date: 'DATE',
+      defence_organisation: 'HASH'
+    }
+  end
+  describe '#perform_later' do
+    subject(:job) do
+      described_class.perform_later(argument_hash)
+    end
+
+    it 'queues a call to update the laa reference' do
+      ActiveJob::Base.queue_adapter = :test
```

Comment:
I would expect this to be in the `config/environments/test.rb` file

Reply:
Moved to the test config

Reply:
0bd65b5 resolves this.